website/docs/Installation.md

```diff
@@ -8,7 +8,7 @@ When not using a docker container, we recommend using a virtual environment to i
 
 You can create a virtual environment with `venv` as below:
 ```bash
-python3 -m venv autogen
+python3 -m venv pyautogen
 source autogen/bin/activate
```

Comment:
you can mention the user can execute deactivate to get out of the python environment.